node/grpc/server_test.go

```diff
@@ -388,6 +498,7 @@ func TestRetrieveChunks(t *testing.T) {
 		QuorumId:        0,
 	})
 	assert.NoError(t, err)
+	assert.Equal(t, pb.ChunkEncoding_GOB, retrievalReply.Encoding)
```

Comment:
why is it gob, since the storeChunkfunction is true?

Reply:
I think in the node/store.go func DecodeChunks, you regard format 0 as ChunkEncoding_GOB. It is confusing since it is sometimes 0 sometimes 2.

Reply:
but anyway, shouldn't the format be 1, since it is requested?

Reply:
By default, `EnableGnarkBundleEncoding` flag is false, so the node will continue to return GOB, even though the underlying data stored in DB is in GNARK.

The ID for protobuf and the enum in golang are different for a reason, added the documentation: https://github.com/Layr-Labs/eigenda/pull/654/commits/797838931b2674ff5784431d2a305ae56035233a